core/types/transaction.go

```diff
@@ -31,6 +31,10 @@ import (
 
 //go:generate gencodec -type txdata -field-override txdataMarshaling -out gen_tx_json.go
 
+const (
+	ETH_COMPATIBLE_TX_NUM_FIELDS = 9
```

Comment:
My understanding is that naming convention is just camelcase, no upcase por constants

Reply:
Fixed